src/modules/dashboard/components/TaskDomains/TaskDomains.tsx

```diff
@@ -85,7 +88,8 @@ const TaskDomains = ({ colonyAddress, domainId, draftId, disabled }: Props) => {
     [colonyAddress],
   );
 
-  const payouts = useSelector(taskPayoutsSelector, [draftId]);
+  // @todo get centralized payouts
```

Comment:
I hope we can find all these. Maybe // FIXME's would be more appropriate (I left a couple as well)